### ISSUE TITLE
1.36.21: FTBFS ( 'AVCodecContext' has not been declared)

### ISSUE THREAD

**Issue by onlyjob** [ZoneMinder/zoneminder#3556]
I have experienced the following FTBFS in Debian "unstable", x86_64:
```
/build/zoneminder-1.36.21+dfsg1/src/zm_packet.h:72:17: error: 'AVCodecContext' has not been declared 
   72 |     int decode( AVCodecContext *ctx ); 
      |                 ^~~~~~~~~~~~~~ 
In file included from /build/zoneminder-1.36.21+dfsg1/src/zm_event.cpp:22: 
```

**Comment by connortechnology** [comment]
This fix should have gone into 1.36.22 or .23 but didn't.  So I'll do 1.36.24 soon like maybe on the weekend.  Let's let .22/.23 settle for a moment.

**Comment by onlyjob** [comment]
1.36.24 builds successfully. Thanks!
